<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 57 16pt &quot;CMU Serif&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="startWidget">
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QWidget" name="chooseFuncWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>531</width>
            <height>151</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>181</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>1.  Выберите функцию:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_1">
             <item>
              <widget class="QRadioButton" name="f1RadioButton">
               <property name="minimumSize">
                <size>
                 <width>531</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>531</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/images/f1.png);
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="f2RadioButton">
               <property name="minimumSize">
                <size>
                 <width>421</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>421</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/images/f2.png);
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="f3RadioButton">
               <property name="minimumSize">
                <size>
                 <width>351</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>351</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/images/f3.png);
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="chooseMethodWidget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QGridLayout" name="nameGridLayout">
             <item row="0" column="3">
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="enterInputDataLabel">
               <property name="minimumSize">
                <size>
                 <width>221</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16</height>
                </size>
               </property>
               <property name="text">
                <string>3.  Введите входные данные:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>150</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="chooseMethodLabel">
               <property name="minimumSize">
                <size>
                 <width>151</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16</height>
                </size>
               </property>
               <property name="text">
                <string>2.  Выберите метод:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="epsGridLayout">
             <item row="0" column="3">
              <widget class="QLineEdit" name="epsLineEdit">
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>341</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>281</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>281</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>-</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Метод дихотомии</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Метод золотого сечения</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Метод Фибоначчи</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Метод парабол</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Комбинированный метод Брента</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="epsLabel">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>21</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>51</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/images/eps.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="deltaGridLayout">
             <item row="0" column="1">
              <widget class="QLabel" name="deltaLabel">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>51</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">image: url(:/images/delta.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>307</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="3">
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="deltaLineEdit">
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>341</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>138</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QStackedWidget" name="errorStackedWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="errorOk">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <spacer name="errorOkHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>715</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="errorFunc">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="errorFuncInfo">
              <property name="minimumSize">
               <size>
                <width>191</width>
                <height>51</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>191</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/images/funcError.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="errorFunchorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>518</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="methodError">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QLabel" name="errorMethodInfo">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>191</width>
                <height>51</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/images/methodError.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="errorMethodhorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>353</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="epsError">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="errorEpsInfo">
              <property name="minimumSize">
               <size>
                <width>381</width>
                <height>41</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>381</width>
                <height>41</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/images/epsError.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="errorEpsHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>328</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="deltaError">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="1">
             <widget class="QLabel" name="errorDeltaInfo">
              <property name="minimumSize">
               <size>
                <width>361</width>
                <height>61</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>361</width>
                <height>61</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">image: url(:/images/deltaError.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="errorDeltaHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>710</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="findMinPushButton">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>31</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 italic 16pt &quot;CMU Serif&quot;;
color: rgb(35, 35, 255);
border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: black;
font: bold 14px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Найти минимум</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainWidget">
       <property name="minimumSize">
        <size>
         <width>754</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="backPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>5</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>29</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 italic 16pt &quot;CMU Serif&quot;;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: black;
font: bold 14px;
padding: 6px;</string>
            </property>
            <property name="text">
             <string>Назад</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="restartPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>5</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>29</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 italic 16pt &quot;CMU Serif&quot;;
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: black;
font: bold 14px;
padding: 6px;</string>
            </property>
            <property name="text">
             <string>Перезапуск</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QCustomPlot" name="plotWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>5</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>736</width>
            <height>430</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(124, 222, 255);</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="infoOutputWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>738</width>
            <height>101</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QListWidget" name="listWidget"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
